roles/KubevirtTemplateValidator/tasks/main.yml

```diff
@@ -1,5 +1,8 @@
 ---
 # tasks file for KubevirtTemplateValidator
+- name: Figure out validator version
+  set_fact:
+    validator_version: "{{ version | default(validator_tag, true) }}"
```

Comment:
You can do this directly in defaults/main.yml or in vars/main.yml, that way it stays local to the role. set-fact sets a global.

Reply:
ok, let's do this in `defaults/main.yml`